common/src/main/java/com/redhat/cloud/notifications/models/SystemSubscriptionProperties.java

```diff
@@ -1,23 +1,17 @@
 package com.redhat.cloud.notifications.models;
 
-import com.fasterxml.jackson.databind.PropertyNamingStrategies;
-import com.fasterxml.jackson.databind.annotation.JsonNaming;
 import jakarta.persistence.Entity;
 import jakarta.persistence.Table;
-import jakarta.validation.constraints.NotNull;
 
 import java.util.Objects;
 import java.util.UUID;
 
 @Entity
 @Table(name = "email_properties")
-@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
 public class SystemSubscriptionProperties extends EndpointProperties {
 
-    @NotNull
```

Comment:
Even if this is a validation annotation, it's fine to remove it because `@NotNull` on a primitive boolean didn't make any sense in the first place.

Reply:
Okay, will not touch those